package.json

```diff
@@ -13,9 +13,7 @@
   "typings": "lib/index.d.ts",
   "files": [
     "bin/",
-    "lib/",
-    ".scripts/",
-    "cached_node_modules/"
+    "lib/"
```

Comment:
if node_modules are in `.tmp/`, not in `lib/`, then this line might be incorrect